notice_communication/__main__.py

```diff
@@ -22,29 +22,45 @@
 os.environ['PATH'] = f"{os.environ['PATH']}:{SCRIPT_PATH}"
 
 def main():
-    with open('/tmp/gcp.key', 'w') as f:
-        f.write(os.getenv('GCP_KEY'))
-    os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = '/tmp/gcp.key'
+    if os.getenv('GCP_KEY'):
+        with open('/tmp/gcp.key', 'w') as f:
+            f.write(os.getenv('GCP_KEY'))
+        os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = '/tmp/gcp.key'
 
     filename = sys.argv[1]
     summary, category, title, slug = get_content_meta(filename)
     url = f"https://bugalert.org/{filename.replace('md', 'html')}"
 
+    # Send Telegram
+    if os.getenv('TELEGRAM_API_KEY'):
+        send_telegram(summary, category, title, url)
+
     if os.getenv('TWITTER_BEARER_TOKEN'):
         twitter = get_twitter_client()
         tweet_summary = summary[:220] if len(summary) > 220 else summary
-        tweet = f"{f'{tweet_summary}...'} {url} #BugAlertNotice"
+        ellipsis = "..." if len(summary) > 220 else ""
+        tweet = f"{f'{tweet_summary}{ellipsis}'} {url} #BugAlertNotice"
```

Comment:
While reviewing this PR I noticed an issue with Twitter formatting that I decided to quickly tackle here.